src/coreclr/vm/interoplibinterface_comwrappers.h

```diff
@@ -0,0 +1,175 @@
+
+extern "C" CLR_BOOL QCALLTYPE TrackerObjectManager_TryRegisterReferenceTrackerManager(_In_ void* manager);
+
+extern "C" void QCALLTYPE TrackerObjectManager_RegisterNativeObjectWrapperCache(_In_ QCall::ObjectHandleOnStack cache);
+
+extern "C" CLR_BOOL QCALLTYPE TrackerObjectManager_IsGlobalPeggingEnabled();
+
+class GlobalComWrappersForMarshalling
+{
+public:
+    static bool TryGetOrCreateComInterfaceForObject(
+        _In_ OBJECTREF instance,
+        _Outptr_ void** wrapperRaw);
+
+    static bool TryGetOrCreateObjectForComInstance(
+        _In_ IUnknown* externalComObject,
+        _In_ INT32 objFromComIPFlags,
+        _Out_ OBJECTREF* objRef);
+};
+
+// Define "manually managed" definitions of the ComWrappers types
+// that are used in diagnostics and during GC.
+class GCHandleSetObject;
+class GCHandleSetEntryObject;
+
+class ManagedObjectWrapperHolderObject : public Object
+{
+    friend class CoreLibBinder;
+private:
+    OBJECTREF _releaser;
+    OBJECTREF _wrappedObject;
+public:
+    DPTR(InteropLib::ABI::ManagedObjectWrapperLayout) ManagedObjectWrapper;
+};
+
+class NativeObjectWrapperObject : public Object
+{
+    friend class CoreLibBinder;
+    OBJECTREF _comWrappers;
+    TADDR _externalComObject;
+    TADDR _inner;
+    CLR_BOOL _aggregatedManagedObjectWrapper;
+    CLR_BOOL _uniqueInstance;
+    OBJECTHANDLE _proxyHandle;
+    OBJECTHANDLE _proxyHandleTrackingResurrection;
+public:
+    OBJECTHANDLE GetProxyHandle() const
+    {
+        return _proxyHandle;
+    }
+
+    TADDR GetExternalComObject() const
+    {
+        return _externalComObject;
+    }
+};
+
+class ReferenceTrackerNativeObjectWrapperObject final : public NativeObjectWrapperObject
+{
+    friend class CoreLibBinder;
+    TADDR _trackerObject;
+    TADDR _contextToken;
+    int _trackerObjectDisconnected;
+    CLR_BOOL _releaseTrackerObject;
+    OBJECTHANDLE _nativeObjectWrapperWeakHandle;
+public:
+    TADDR GetTrackerObject() const
+    {
+        return (_trackerObject == (TADDR)nullptr || _trackerObjectDisconnected == 1) ? (TADDR)nullptr : _trackerObject;
+    }
+};
+
+#ifdef USE_CHECKED_OBJECTREFS
+using MOWHOLDERREF = REF<ManagedObjectWrapperHolderObject>;
+using NATIVEOBJECTWRAPPERREF = REF<NativeObjectWrapperObject>;
+using REFTRACKEROBJECTWRAPPERREF = REF<ReferenceTrackerNativeObjectWrapperObject>;
+using HANDLESETENTRYREF = REF<GCHandleSetEntryObject>;
+using HANDLESETREF = REF<GCHandleSetObject>;
+#else
+using MOWHOLDERREF = DPTR(ManagedObjectWrapperHolderObject);
+using NATIVEOBJECTWRAPPERREF = DPTR(NativeObjectWrapperObject);
+using REFTRACKEROBJECTWRAPPERREF = DPTR(ReferenceTrackerNativeObjectWrapperObject);
+using HANDLESETENTRYREF = DPTR(GCHandleSetEntryObject);
+using HANDLESETREF = DPTR(GCHandleSetObject);
+#endif
+
+class GCHandleSetEntryObject final : public Object
+{
+    friend class CoreLibBinder;
+    public:
+    HANDLESETENTRYREF m_next;
+    OBJECTHANDLE m_handle;
```

Comment:
```suggestion
    HANDLESETENTRYREF _next;
    OBJECTHANDLE _handle;
```
Delete the `m_` prefix while you are on it to follow the libraries coding convention?